gremlin-client/src/conversion.rs

```diff
@@ -205,3 +211,12 @@ impl_borrow_from_gvalue!(i64, GValue::Int64);
 impl_borrow_from_gvalue!(uuid::Uuid, GValue::Uuid);
 impl_borrow_from_gvalue!(chrono::DateTime<chrono::Utc>, GValue::Date);
 impl_borrow_from_gvalue!(bool, GValue::Bool);
+
+#[test]
```

Comment:
@wolf4ood I'm still learning Rust so this may not be the best place for this. From what I've learned so far it seems unit tests are added like this. But if you want it else where let me know, happy to move it.

Reply:
Hi @criminosis 

i usually place unit tests in the same file. If they grow too much we could move it in a separate file via `#[cfg(test)]` + `#[path = ".."]`